type-tests/index.d.ts

```diff
@@ -0,0 +1,3 @@
+// TypeScript Version: 3.5
+
+export * from '../src';
```

Comment:
Why the re-export here and below? Maybe a comment explaining it somewhere (docs/README/etc.)?